.gitignore

```diff
@@ -1,116 +1,2 @@
 target/
 .idea
-
```

Comment:
Is there a reason you are removing these files from the gitignore? This means you want they files tracked on the local repository. Is there a reason for this? Are these files no longer present/generated?

Reply:
On Github I got prompted an automatic merge button which deleted a bunch of these files.

Reply:
Files still show as deleted, please make sure you've pushed any merge commits.

Reply:
Thanks for helping me rebase @zstewar1